### DISCUSSION TITLE
Creating read-only users

### DISCUSSION THREAD

**Discussion by kiwioverseas** [planetscale/discussion#221] [Q&A] [answered]
How can read-only users be added to the database? CREATE USER statement returns an error.

**Comment by jgreet** [comment, marked as answer]
This is now available! https://docs.planetscale.com/concepts/password-roles

~Sorry, it isn't possible to create read-only database credentials yet.~

~Database usernames and passwords are created through the [connection string interface in the web app](https://docs.planetscale.com/concepts/connection-strings) or via`pscale password` using the CLI client.  `CREATE USER` won't work on the MySQL command-line with a PlanetScale database. These credentials all have write access at the moment but more options are coming.~

**Reply by jgreet** [reply]
You can now create read only connection strings in the web interface:
https://docs.planetscale.com/concepts/password-roles

**Reply by will-holley** [reply]
Is selecting a subset of tables as read-only currently supported, @jgreet?

**Reply by jgreet** [reply]
It is not. Permissions are still set at the credential level and apply to all of your tables.

**Reply by will-holley** [reply]
Thanks for the quick response, John.  This may be a non-starter for me, unfortunately.  For context, I'm storing documents that contain Personally Identifiable Information (PII) and would like to redact and store the PII values in a separate table with more restrictive permissions. Any workarounds or potential reframings of the architecture?
